Bottom: 991e4f477afe0f5f7db38751005b2ba7e991c213
Top:    2191f0dcdf934cc89c175e7e3883d00f27c41606
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-08 20:41:29 +0000

Refresh of try-debugging-draw-speed

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index fbd9ecf..b3c525d 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -1568,7 +1568,7 @@ Benchmark (int argc, char **argv, Coord x, Coord y)
     }
   while (end - start < 10);
 
-  printf (_("%g redraws per second\n"), i / 10.0);
+  printf (_("%g redraws per second\n"), (double)i / (double)(end-start));
 
   return 0;
 }
